jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBStatement.java

```diff
@@ -322,31 +322,32 @@ private boolean executeSQL(String sql) throws TException, SQLException {
     TSStatus execResp = client.executeBatchStatement(execReq);
     int[] result = new int[batchSQLList.size()];
     boolean allSuccess = true;
-    String message = "\n";
+    StringBuilder message = new StringBuilder("\n");
     for (int i = 0; i < result.length; i++) {
       if (execResp.getCode() == TSStatusCode.MULTIPLE_ERROR.getStatusCode()) {
         result[i] = execResp.getSubStatus().get(i).code;
         if (result[i] != TSStatusCode.SUCCESS_STATUS.getStatusCode()
             && result[i] != TSStatusCode.NEED_REDIRECTION.getStatusCode()) {
           allSuccess = false;
-          message +=
+          message.append(
               execResp.getSubStatus().get(i).message
                   + " for SQL: \""
                   + batchSQLList.get(i)
                   + "\""
-                  + "\n";
+                  + "\n");
```

Comment:
message.append(execResp.getSubStatus().get(i).message).append(" for SQL: \"")
              .append(batchSQLList.get(i)).append("\"").append("\n");